src/method.jl

```diff
@@ -214,9 +214,9 @@ function solve!(
     rem_eval = max_eval - neval_obj(nlp)
     solved = normgp ≤ ϵd && normcx ≤ ϵp
     jtprod!(nlp, al_nlp.x, al_nlp.cx, solver.Jtv)
-    infeasible = al_nlp.μ > 1 / eps(T) && norm(solver.Jtv) < √ϵp * normcx
-    tired =
-      iter > max_iter || el_time > max_time || neval_obj(nlp) > max_eval || al_nlp.μ > 1 / eps(T)
+    penalty_too_large = al_nlp.μ > 1 / eps(T)
+    infeasible = penalty_too_large && norm(solver.Jtv) < √ϵp * normcx
+    tired = iter > max_iter || el_time > max_time || neval_obj(nlp) > max_eval
```

Comment:
or penalty too large?

Reply:
What do you mean? So, I get rid of `infeasible`?

Reply:
nvm